keras/models.py

```diff
@@ -28,33 +28,15 @@
     h5py = None
 
 
-def save_model(model, filepath, overwrite=True, include_optimizer=True):
-    """Save a model to a HDF5 file.
-
-    The saved model contains:
-        - the model's configuration (topology)
-        - the model's weights
-        - the model's optimizer's state (if any)
-
-    Thus the saved model can be reinstantiated in
-    the exact same state, without any of the code
-    used for model definition or training.
+def _save_model(model, f, include_optimizer=True):
```

Comment:
This is a private method, for safety, better not pass a default value for `include_optimizer`. Users shouldn't be using it, and when working on Keras itself it is easy to forget to pass it.

Reply:
Agree. Will do it here and on the `_load_model`